src/Randomizer.SMZ3.Tracking/VoiceCommands/GrammarException.cs

```diff
@@ -52,6 +52,7 @@ public GrammarException(string? message, Exception? innerException)
         /// The StreamingContext that contains contextual information about the
         /// source or destination.
         /// </param>
+        [Obsolete("Serialization constructors are deprecated in .NET8+", DiagnosticId = "SYSLIB0051")]
         protected GrammarException(SerializationInfo info, StreamingContext context)
```

Comment:
I don't think we even use this. I'm pretty sure this just came 'free' with the exception snippet, so we could just remove it outright.

Reply:
Yeah, I waffled on removing it. I think both exceptions we only use one of the constructors. I'll clean them up.